### PR TITLE
Updated to .net 8

### PR DESCRIPTION
**Pull request by danielaase1337**
Upgraded the project to .net8. 
No other changes, than version upgrade and file/assembly verions to 8.0

### CODE REVIEW

**Comment by andyward** [comment]
Hi Daniel - thanks for contributing

Looks like the [Azure Pipeline buildfile](https://github.com/xBimTeam/XbimPropertySets/blob/63d98aa0e9e897162746caabd101f61e758364c3/azure-pipelines.yml#L57) needs updating too? 

Out of interest, why do you need to update to net8 for this simple library? This will make it incompatible for anyone on net6/net7 or netframework. I guess there may be reasons, but we should really multi-target frameworks so that consumers on older runtimes can use this project. If you update we can push through.

File: Xbim.Properties/Xbim.Properties.csproj [hunk 1]
```diff
@@ -1,114 +1,115 @@
...
-		<Title>xbim.PropertySets</Title>
-		<Description>Programmatic access to BuildingSmart Properties and Psets for IFC schemas, and related Data Dictionaries</Description>
-	</PropertyGroup>
-	<PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|netstandard2.0|AnyCPU'">
-	  <DefineConstants>DEBUG;TRACE</DefineConstants>
-	</PropertyGroup>
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <RestorePackages>true</RestorePackages>
+    <TargetFrameworks>net8.0</TargetFrameworks>
```
Comment (andyward) [inline Xbim.Properties/Xbim.Properties.csproj:5]: I think this should be
`<TargetFrameworks>netstandard2.0;netstandard2.1;net8.0</TargetFrameworks>`

**Comment by danielaase1337** [comment]
At first I thought I needed an updated version to get rid of some extra files in connection with use in Revit 2025.

Checked the dependencies a little more closely, and see that there won't be a problem using the .net6 version.

At closer look, just skip the PR and keep it.

**Comment by andyward** [comment]
We've seen issues with clashing versions of things like Microsoft.Extensions.Logging.Abstractions from other Revit addins before. Will be updating to Revit 2025 soon so can report back if we have issues with our addins

**Pull request closed without merging**